docs/AppBar.md

```diff
@@ -26,11 +42,11 @@ The `<pxb-app-bar>` component is a wrapper around the `<mat-toolbar>` that can b
 | scrollContainerClassName    | Class name, index number of scrollable element               | `{ name: string, index: number }`            | no       |                                        |
 | scrollContainerId           | Id of the scrollable element                                 | `string`                                     | no       |                                        |
 | scrollThreshold             | Distance in pixels to scroll before collapsing toolbar       | `number`                                     | no       | expandedHeight - collapsedHeight       |
-| variant                     | Behavior mode of the App Bar                                 | `'expanded'` \| `'collapsed'` \| `'snap'`    | no       | 'snap'                                 |
+| variant                     | Behavior of the App Bar                                 | `'expanded'` \| `'collapsed'` \| `'snap'`    | no       | 'snap'                                 |
 
 </div>
 
-> For the `snap` mode to respond correctly to scroll events, the `scrollContainerElement`, `scrollContainerClassName`, or `scrollContainerId` should be provided. 
+> For the `snap` variant to respond correctly to scroll events, the `scrollContainerElement`, `scrollContainerClassName`, or `scrollContainerId` should be provided. 
 
 
 The following child element is projected into `<pxb-app-bar>`:
```

Comment:
Following Angular Material's tradition, there should be `[ ]` around the selector name. See https://pxblue-components.github.io/angular-dev/?path=/info/components-empty-state--readme#api 

(Would be nice if you can fix some other readmes to line up with this)